core/src/main/resources/lib/platform/zephyr/boards/nrf52dk_nrf52832.overlay

```diff
@@ -0,0 +1,3 @@
+&timer1 {
```

Comment:
Should this change be included in this PR, or another Zephyr-specific PR?

Reply:
This is some changes needed for Zephyr. They are already in master. The best would be to bring this `static-schedule` branch up-to-date with master first. Then I wouldnt need that change

Reply:
Ok, sounds good. I will merge bring the `static-schedule` branch up-to-date.

Reply:
The `static-schedule` branch has been updated.